.scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+version=2.5.2 
+maxColumn = 80
```

Comment:
80 is too conservative. I think 120 would be better. And if that becomes too much, we can tighten it later.